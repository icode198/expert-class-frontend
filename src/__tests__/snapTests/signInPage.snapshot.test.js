import React from 'react';
import { cleanup } from '@testing-library/react';
import ReactRouter from 'react-router';
import { renderWithRedux } from '../../tests/test-utils';
import SignInPage from '../../pages/signInPage/signInPage';

afterEach(cleanup);

describe('SignIn Page', () => {
  test('Renders the page', () => {
    jest.spyOn(ReactRouter, 'useLocation').mockReturnValue({ state: { from: 'classes' } });

    const app = renderWithRedux(<SignInPage />).toJSON();

    expect(app).toMatchSnapshot();
  });
});
