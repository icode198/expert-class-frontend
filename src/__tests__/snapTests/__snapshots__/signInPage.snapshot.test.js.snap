// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn Page Renders the page 1`] = `
<div
  className="page-container"
>
  <div
    className="innerContainer"
  >
    <h2
      className="title"
    >
      Sign In
    </h2>
    <form
      className="signInForm"
      onSubmit={[Function]}
    >
      <input
        name="username"
        onChange={[Function]}
        placeholder="username"
        required={true}
        type="text"
        value=""
      />
      <input
        className="button-white"
        type="submit"
        value="Sign in"
      />
    </form>
  </div>
</div>
`;
